### ISSUE TITLE
Is there a way to limit the OCR and Inpainting function to just focus on text bubbles and text boxes only during automatic run

### ISSUE THREAD

**Issue by himouto** [dmMaze/BallonsTranslator#265]
I want to exclude the SFX or background sound effect characters (characters with no text bubbles) from getting recognized by OCR + Inpainting because sometimes the automatic inpainting messes up and distorts parts of the drawing - I know this can be done through manual text selection but I hope there's a way to achieve this using the automatic run

**Comment by dmMaze** [comment]
No, it cannot be achieved directly for now, alternately if you're running the dev branch:
1. check the `Delete and restore region where OCR returns empty string.` under the OCR config
2. Edit->Keyword substitution for OCR, add some regexp to replace those SFX recognized by OCR to empty string

**Comment by Alexamenus** [comment]
this is still an issue, bump. we need a mechanism to skip inpainting on boxes the box detector marked that isn't actually recognizable text for the ocr module,, causing lot's of removed details and empty onomatopoeia boxes as well as many empty boxes within the UI.
